conan/tools/gnu/autotools.py

```diff
@@ -15,17 +17,39 @@ def __init__(self, conanfile, namespace=None):
                                                      namespace=namespace)
         self._configure_args = toolchain_file_content.get("configure_args")
         self._make_args = toolchain_file_content.get("make_args")
+        self.default_configure_install_args = True
+        self.build_script_folder = None
 
     def configure(self, build_script_folder=None):
         """
         http://jingfenghanmax.blogspot.com.es/2010/09/configure-with-host-target-and-build.html
         https://gcc.gnu.org/onlinedocs/gccint/Configure-Terms.html
         """
-        source = self._conanfile.source_folder
-        if build_script_folder:
-            source = os.path.join(self._conanfile.source_folder, build_script_folder)
+        configure_folder = os.path.join(self._conanfile.source_folder, build_script_folder) \
+            if build_script_folder else self._conanfile.source_folder
 
-        configure_cmd = "{}/configure".format(source)
+        self.build_script_folder = self.build_script_folder or os.path.join(configure_folder)
+        configure_args = []
+        if self.default_configure_install_args and self._conanfile.package_folder:
+            def _get_cpp_info_value(name):
+                # Why not taking cpp.build? because this variables are used by the "cmake install"
+                # that correspond to the package folder (even if the root is the build directory)
+                elements = getattr(self._conanfile.cpp.package, name)
+                return elements[0] if elements else None
```

Comment:
Isn't this defining ``--bindir=${prefix}/None`` for some cases? If not, then it should be an assert?

Reply:
I can avoid that omitting the not defined ones, but in general they should be defined with the default value?